ports/stm32/irq.h

```diff
@@ -39,7 +39,7 @@
 #define IRQ_ENABLE_STATS (0)
 
 #if IRQ_ENABLE_STATS
-extern uint32_t irq_stats[FPU_IRQn + 1];
+extern uint32_t irq_stats[256];
```

Comment:
This will need a corresponding change in irq.c, and then it'll be good to define a constant here for the number of entries, eg `IRQ_STATS_MAX`.  Could make that value 256 for H7 and 128 for all other MCUs.

Reply:
Do you want this constant defined here in irq.h ?

Reply:
Yes, I think it needs to be